Persian language for OpenToonz website

pikhosh opened the pull request:
Hello! The Persian translation of the OpenToonz website was done to a large extent 😀

Things I did:

- Translate lots of text from all sections
- Add Persian language button to other pages as well as English language button to Persian page
- Modify CSS to add new buttons
- Correction of metadata problems (information? I do not know what to say to it) in HTML files in English and Persian
- Add the Vazir font to the Persian website (if it works 😂)
- Remove the Spanish button from the UseCase page of all languages (Because that page was not available in Spanish and the button was kind of useless)

Things other friends should do (because I can not 🙄):

- Right to left Persian website correctly
- Add Persian button on Spanish page (Spanish website codes look different and the buttons are images)

Things I do:

- Complete Persian translations
- Translate images into Persian

Overall, I hope the work I did will be useful 😅

_(I apologize for the problems in this text.)_

pikhosh committed:
Persian language for OpenToonz website

shun-iwasawa commented:
Thank you @piki-shabpareh ! I'll check it within days.

To someone who is interested: You can preview the Persian webpage by this PR [here](https://htmlpreview.github.io/?https://github.com/piki-shabpareh/opentoonz.github.io/blob/Persian/fa/index.html).

shun-iwasawa committed:
right to left layout

pikhosh committed:
Persian translation of images

Persian translation of images was also done!
With two small modifications:
In the Spanish "index" file; The CSS filename was misspelled.
And I shortened the persian translation of "FAQ" because of the bad look in the style of the theme.

pikhosh committed:
Fix RTL layout problems

Some changes in the distance between the elements
LTR footer text
Apply font to download pages

pikhosh committed:
Fix Facebook and Twitter buttons problems in Mobile Layout

Correct the distance between the elements in the Facebook and Twitter buttons

pikhosh committed:
Remove useless spaces in RTL layout

A few small deletions!

pikhosh committed:
Update index.html

Add a few more Persian translations

pikhosh commented:
@shun-iwasawa Hello! For now, I have solved the problems of the right-to-left theme, as well as translated the remaining images and some texts into Persian.

But translations of parts of the "Index", "UseCase" and the text "Terms of Use" remain

I do not know when I will have time to translate these parts, so if you have no problem, the Persian version will be published with these changes.

shun-iwasawa commented:
@pikhosh  Thank you the great work. LGTM
